### DISCUSSION TITLE
`useCallback` in readme example

### DISCUSSION THREAD

**Discussion by mkosir** [ValentinH/react-easy-crop#485] [General]
Hey 👋 
thanks for an awesome library 🚀 
Just wondering in [basic example](https://github.com/ValentinH/react-easy-crop#basic-usage) how much sense does it make to wrap `onCropComplete` into `useCallback`. Maybe we could consider removing it, or add some explanation on what (and when) benefits it brings, wdyt?

**Comment by ValentinH** [comment]
Indeed, you are right; it's not required.

 I made it this way when I was planning to rewrite this library using hooks. In that case, I would have needed a `useEffect` using the `onCropComplete` as a dependency. Teaching the users to wrap it with `useCallback` would have made it safer.
However, I know have some tricks to avoid to use it as a dependency so it would be safe. Anyway, I will not rewrite to hooks anytime soon, last time I tried, it was way too hard. 🙈

**Reply by ValentinH** [reply]
I've updated all sandboxes and the Readme 🚀

**Reply by mkosir** [reply]
Awesome thx 🚀
